packages/CopyInput/src/CopyInput.js

```diff
@@ -46,12 +48,18 @@ function CopyInput(props) {
   }
 
   return (
-    <sc.CopyInput data-pka-anchor="copy-input" {...moreProps}>
-      <Input ref={inputRef} defaultValue={value} isReadOnly={isReadOnly} {...extendedInputProps} />
+    <sc.CopyInput data-pka-anchor="copy-input" hasDefaultButtonBorder={!hasInputContainer} {...moreProps}>
+      {hasValueContainer ? <sc.Value>{value}</sc.Value> : null}
+      {!hasInputContainer ? (
+        <sc.HiddenInput ref={inputRef} defaultValue={value} />
+      ) : (
+        <Input isReadOnly={isReadOnly} ref={inputRef} defaultValue={value} {...extendedInputProps} />
+      )}
```

Comment:
Very small point, not worth re-rerunning semaphore for, but usually it's a good practice (a bit easier to read) a ternary expression if you don't include the `!` in the condition, basically reverse these cases.